tests/phpunit/integration/api/term-meta/fixtures/test-field-outout-html.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ *  Test field output HTML for use with render_fields method test.
+ *
+ * @package Beans\Framework\Tests\Integration\API\Term_Meta\Fixtures
+ *
+ * @since 1.5.0
+ */
+
+return <<<FIELDSHTML
+<tr class="form-field">
+	<th scope="row">
+			</th>
+	<td>
+		<div class="bs-field-wrap bs-radio term_meta"><div class="bs-field-inside"><div class="bs-field bs-radio">
+<fieldset class="bs-field-fieldset">
+	<legend class="bs-field-legend">Layout</legend>
+		<label class="" for="beans_layout_test_default_fallback">
+		
+	<input id="beans_layout_test_default_fallback" type="radio" name="beans_fields[beans_layout_test]" value="default_fallback" checked='checked' /> 	Use Default Layout		</label>
+		<label class="bs-has-image" for="beans_layout_test_c">
+					<span class="screen-reader-text">Option for c</span>
+			<img src="/app/public/wp-content/themes/tm-beanslib/admin/assets/images/layouts/c.png" alt="Option for c" />
```

Comment:
The pathing will be different on Travis and different servers.  Therefore, you need to use a variable for the site's URL here and in each spot where you are validating the HTML.